src/app/register/page.tsx

```diff
@@ -2,95 +2,105 @@ import { Input } from "@/components/ui/input";
 import { Label } from "@/components/ui/label";
 import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
 import { registerAgent } from "../actions";
-import { SubmitButton } from "./submit-button";
+import { Button } from "@/components/ui/button";
+import Link from "next/link";
+import { SubmitButton } from "@/components/SubmitButton";
 
 export default function RegistrationForm() {
```

Comment:
The code diff is a bit hard to read, what was added apart from moving the submit button and the back link?